ingress-lb/kontena.yml

```diff
@@ -29,9 +29,7 @@ services:
     image: kontena/lb:latest
     ports:
       - 80:80
-      # {% if lb_certs %}
       - 443:443
```

Comment:
I'm pretty sure the `{% if lb_certs %}` conditional was true before if you just hit enter on the `lb_certs` prompt... If I `kontena stack install kontena/ingress-lb` without selecting any (vault) certs, then the service will have port 443 exposed?

Reply:
Yes, imho it's better to open this port always so that LE tls-sni-01 can work properly.

Reply:
Yes. I'm just saying that I think the port 443 was already open, even before this change :)